3_javascript/javascript1/week2/homework2ndweek/fullname.html

```diff
@@ -0,0 +1,24 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Flight booking</title>
+  </head>
+  <body>
+    <script>
+      const firstname1 = getFullName("Monika", "Marciniak");
+      const firstname2 = getFullName("John", "Smith", false);
+
+      function getFullName(firstName, surName, isWoman = true) {
```

Comment:
is woman is another case, what you should implement first is `useFormalName`, so when `useFormalName= true` then you add Lord or Lady depending on the gender.

Reply:
I dont exactly understand whats wrong, i think it works.